### PR TITLE
Make Invoice List Great Again

### PR DESCRIPTION
**Pull request by Kukks**
This PR adds a more user-friendly UI to the invoice list for querying. 

* The button at the top-right allows you to collapse the filters in order to conserve space. 
* Fallback to old search if js not enabled

image

image

### CODE REVIEW

File: BTCPayServer/BTCPayServer.csproj [hunk 1]
```diff
@@ -190,4 +190,10 @@
 	    <Pack>$(IncludeRazorContentInPack)</Pack>
 	  </Content>
 	</ItemGroup>
+
+	<ItemGroup>
+	  <_ContentIncludedByDefault Remove="wwwroot\invoice-filter\helpers\math.js" />
+	  <_ContentIncludedByDefault Remove="wwwroot\payment-request\helpers\math.js" />
+	  <_ContentIncludedByDefault Remove="wwwroot\invoice-filter\styles\main.css" />
```
Comment (NicolasDorier) [inline BTCPayServer/BTCPayServer.csproj]: what is this?

Comment (Kukks) [reply BTCPayServer/BTCPayServer.csproj]: Not sure, looks like IDE junk

**Comment by NicolasDorier** [comment]
`noExponents` is used at other places, but you deleted it.

**Comment by Kukks** [comment]
@NicolasDorier 
> `noExponents` is used at other places, but you deleted it.

No, I moved it to a common file and referenced it in the bundles because before the js was duplicated in both apps

**Comment by NicolasDorier** [comment]
ok good for me maybe @pavlenex wants to bitch about something as well?

**Comment by rockstardev** [comment]
@pavlenex trying to save you from UI horror ;) let me give a try on this.

**Comment by rockstardev** [comment]
OK, I've started handling stuff me and @pavlenex agreed on... first PR is #790.

Change are conflicting with this PR so I've checking out `Kukks:better-invoice-list` branch locally and will see to merge Javascript as I go. I'll close this PR and will update @Kukks over Mattermost as I go. <3

**Pull request closed without merging**